### PR TITLE
[DO-2232]Changed logic of SelectColumns

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -381,6 +381,7 @@
   "topCompanies": "TOP companies",
   "topKveds": "TOP NACE",
   "totalRecords": "Total records",
+  "TurnOnAll": "Turn on all",
```
Comment (roman-tiukh) [inline src/locales/en/translation.json:384]:
```suggestion
  "turnOnAll": "Turn on all",
```

File: src/components/table/SelectColumns.jsx [hunk 2]
```diff
@@ -1,68 +1,116 @@
...
+    setValues(newValues);
+  };
+
+  const handleCancelClick = (e) => {
+    setValues([...selectedColumnsNames]);
+  };
+
+  const handleApplyClick = (e) => {
+    dispatch(setSelectedColumns(tableUrl, values));
   };
```
Comment (roman-tiukh) [inline src/components/table/SelectColumns.jsx:42]: this functions can be inline in jsx

File: src/locales/en/translation.json (hunk 1 above)
Comment (IegorSoboliev) [reply src/locales/en/translation.json]: thank you, fixed!

File: src/components/table/SelectColumns.jsx (hunk 2 above)
Comment (IegorSoboliev) [reply src/components/table/SelectColumns.jsx]: done

**Review by AlenaYanish — APPROVED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**